### PR TITLE
Initial setup with homescreen

### LINKED ISSUE #1: Initial project setup and home view
Set up project with home view that has a category card grid layout with some dummy content.

### LINKED ISSUE #2: Home view carousel
Add carousel (horizontally scrolling view) to top of home view, with dummy content.

### LINKED ISSUE #3: Home view top bar
Add settings drop-down and search bar, not implemented, to top of home view

### PR DESCRIPTION
**Pull request by flibbertigibbet**
## Overview

Initial app setup with a single activity that loads test images into a carousel at the top, with a scrolling grid of test images below. Adds translucent toolbar with options menu. Clicking on toolbar, carousel, or grid item logs what was clicked. Wireframe [here](https://marvelapp.com/4c295bb/screen/37236996).

## Demo

Pixel 2 emulator, API level 27:

pixel_2_emulator_home

Toolbar open:

pixel_2_emulator_toolbar_menu

Nexus One emulator, running API 19 (minimum supported):

nexus_one_api_19_home

## Notes

Forked the carousel library to support targeting the latest API version and opened an upstream PR: sayyam/carouselview#83. If not merged upstream, should move fork from personal to company repository. Was able to install library via [jitpack](https://jitpack.io/#flibbertigibbet/carouselview/1.0.5) without any further configuration.

Went with the default GitHub/Android Studio `gitignore` configuration, but there still seem to be some included `.idea` files that may be better ignored. If just checking out and running the project for review results in a diff, those files should probably be ignored as well.

## Testing

 - Run in an emulator on API level 19 - 27
 - Carousel should autoplay
 - Should also be able to drag carousel to move forward or back
 - Should be able to scroll through images grid
 - Should be able to open toolbar overflow menu from upper right dots
 - Clicking on an item (carousel, grid, or toolbar) should log what was selected

Closes #1 
Closes #2 
Closes #3

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/cac-tripplanner-android.iml" filepath="$PROJECT_DIR$/cac-tripplanner-android.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (maurizi) [inline .idea/modules.xml]: All the other `.idea` files were unmodified, but this one was dirtied by importing it into Android Studio, and should be added to `.gitignore`.

**Comment by maurizi** [comment]
Reported lint:
screenshot from 2018-02-08 14-49-47

**Comment by flibbertigibbet** [comment]
Good to note the linter output; should definitely review that before publishing.

File: .idea/modules.xml (hunk 1 above)
Comment (flibbertigibbet) [reply .idea/modules.xml]: Whoops, just saw this note after merge. Good to know, will add to `.gitignore` in a future PR.

**Pull request merged**